src/dropin.js

```diff
@@ -357,6 +351,33 @@ Dropin.prototype.clearSelectedPaymentMethod = function () {
   this._model.removeActivePaymentMethod();
 };
 
+Dropin.prototype._setUpDataCollector = function () {
+  var config = assign({}, this._merchantConfiguration.dataCollector, {client: this._client});
+
+  this._model.asyncDependencyStarting();
+
+  dataCollector.create(config).then(function (instance) {
+    this._deviceData = instance.deviceData;
+  }.bind(this)).catch(function (err) {
+    console.log('Data Collector failed to set up', err);
```

Comment:
Not sure what the right thing to do here is. I've elected to log to the console that data collector couldn't be set up and continue on as if the dependency didn't fail (since there's no UI updates to make to the Drop-in for this failure).

Open to suggestions for a better way to handle this.